examples/onnxflow/lgbmclassifier.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
Please add this file to the test suite here: https://github.com/groq/mlagility/blob/c683577f3284ec9665d8916494e0cef1536ac47c/.github/workflows/test_onnxflow.yml#LL48C17-L48C17